README.md

```diff
@@ -1,2 +1,70 @@
-# labs-contracts
-Contracts made by Gnosis Labs team.
+# Gnosis Labs Contracts
+
+Repository holding the contracts made by Gnosis Labs team.
+
+## Implemented contracts
+
+### Omen ThumbnailMapping
+
+Contract used to store prediction market's address to IPFS hash of an image displayed on Omen 2.0
+
+## Set up
+
+The repository uses [Foundry](https://book.getfoundry.sh/).
+
+### Installation
+
+See installation instructions on https://book.getfoundry.sh/getting-started/installation.
+
+### Build
+
+```shell
+forge build
+```
+
+### Test
+
+```shell
+forge test
+```
+
+### Format
+
+```shell
+forge fmt
+```
+
+### Gas Snapshots
+
+```shell
+forge snapshot
+```
+
+### Anvil
+
+```shell
+anvil
+```
+
+### Deploy
+
+```shell
+forge create --gas-limit 10000000 --rpc-url <your_rpc_url>  --private-key <your_private_key> OmenThumbnailMapping
```

Comment:
Please use the --verify param to automatically verify the contract. It makes interaction with it much easier afterwards.

Reply:
Registration on gnosisscan is broken, I'll verify it later manually, but added into readme